### PR TITLE
fix nil pointer when adding a due date 

### PR DESCRIPTION
**Pull request by adelowo**
I was trying to pull #5568 to test it locally. But on trying to add a ___due date___,
 I kept on getting "The due date is invalid or out of range. Please use the format 'yyyy-mm-dd'."
 Looked at my terminal and it was a panic. 

Commit I am running locally : a06ae0bdc74405bcf769f3f9fe28296b8e7cdda4

Log --> https://gist.github.com/adelowo/d74a006e9d77d6be6fa41a50a5e09008

### CODE REVIEW

**Review by kolaente — APPROVED** [review]

File: models/issue_comment.go [hunk 1]
```diff
@@ -708,6 +708,12 @@ func createDeadlineComment(e *xorm.Session, doer *User, issue *Issue, newDeadlin
 		content = newDeadlineUnix.Format("2006-01-02") + "|" + issue.DeadlineUnix.Format("2006-01-02")
 	}
 
+	if issue.Repo == nil {
```
Comment (lunny) [inline models/issue_comment.go]: This line is unnecessary since LoadRepo has done that.

**Review by lunny — CHANGES_REQUESTED** [review]

**Review by zeripath — APPROVED** [review]

File: models/issue_comment.go (hunk 1 above)
Comment (rvillablanca) [reply models/issue_comment.go]: Changes have been made so is possible to merge this PR ?

**Comment by rvillablanca** [comment]
Changes have been made so is possible to merge this PR ?

**Comment by zeripath** [comment]
@rvillablanca - I think @lunny needs to accept your changes. He's on Chinese time (I think) (GMT+8) it's not quite 6am there currently.

**Comment by rvillablanca** [comment]
Thanks @zeripath, these are not my changes but I need them to make my changes over master branch and not over v1.6.2 tag.

**Comment by lafriks** [comment]
@lunny need your approval

**Review by lunny — APPROVED** [review]

**Pull request merged**